test/functional/commands/touch-action-e2e-specs.js

```diff
@@ -77,19 +78,24 @@ describe('apidemo - touch', function () {
 
   describe('mobile: scrollBackTo', function () {
     let driver;
-    before(async () => {
+    before(async function () {
+      if (await isArmEmu()) {
+        // arm emulators are too slow and android's scrollToBeginning is broken
+        // on them for some reason
```

Comment:
Are you sure this is an ARM problem? Did you tried to digging into this issue?

Reply:
i did try digging in. the issue is that android thinks it is at the top of the view and returns as such, even when this is not the case. i could only reproduce it on arm emus. i don't know if it has anything to do with arm itself. probably not. i wonder if it has to do with the slowness causing parts of the gesture to not be recognized, or cause some other confusion.

in short, i don't fully understand the root cause, but given that it only seems to affect a class of emus no one actually uses, except us in CI, i felt fine skipping.

Reply:
This is strange that you can reproduce this issue on arm emu only, because all scrolling stuff are currently broken on uia2 - appium/appium#9328, #101. Can you repro this issue without ```NotificationListener```?

Reply:
interesting, @vmaxim. i didn't realize uiauto2 could steal notifications from other listeners. that may indeed be what's happening. either way it seems unrelated to this PR, for which quarantining the test for environments it's broken in is reasonable, as long as we don't forget to fix the underlying problem and unquarantine this test when that is resolved.